Program.cs

```diff
@@ -100,9 +98,11 @@ public static void RunSample(IAzure azure)
                 {
                     azure.ServiceBusNamespaces.DeleteById(serviceBusNamespace.Id);
                 }
-                catch (Exception)
+                catch (Exception e)
                 {
+                    Utilities.Log("Unexpected error occured: " + e.Message);

```

Comment:
All of these `Log` calls resolve to a simple `Console.WriteLine`  the indirection makes it less clear what is going on and is unnecessary.

Reply:
Resolved